cgi-bin/LJ/Event/RemovedFromCircle.pm

```diff
@@ -103,13 +105,14 @@ sub _as_email {
             . $self->format_options(
             $is_html, undef, $vars,
             {
+                'esn.show_unsubscribed_journal' => [ 0, $journal_url ],
                 'esn.remove_watch' => [
-                    !$u->watches( $self->fromuser ) ? 0 : 1,
+                    !$u->watches( $self->fromuser ) ? 1 : 2,
                     "$LJ::SITEROOT/circle/$postername/edit"
```

Comment:
So looking at how format_options works: if the first element here is 0, it doesn't show at all, so this should actually be `[ 1, $journal_url ]` and `!$u->watches( $self->fromuser ) ? 0 : 2` in order to do what you want. (Same also for trusts.)